src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -301,6 +311,19 @@ const TreeItem: React.FC<TreeItemProps> = props => {
         ]
       : []),
     {key: 'divider-4', type: 'divider'},
+    ...(remotePath
```

Comment:
We should check if the git repo exists and if the remote is a GitHub repo, otherwise, the option should not be available ( hidden )

Reply:
I think this ...( syntax doing it if remote path is there its an none empty array and ifts not it will not show anything

Reply:
But what if the `remotePath` is a GitLab one?

Reply:
oh you are right I need to handle inside getGitTopLevel function, not sure if it returns null in case its not github will test this 👍

Reply:
Pushed an update for this 👍